pyproject.toml

```diff
@@ -1,64 +1,58 @@
-[build-system]
-requires = [
-    "poetry-core>=1.0.0", "setuptools>=41.0.0", "wheel", "cffi"]
-build-backend = "poetry.core.masonry.api"
-
-[tool.poetry.build]
-script = "pyreaddbc/_build_readdbc.py"
-generate-setup-file = false
-
-[tool.poetry.group.dev.dependencies]
-pandas = "^2.1.0"
-
-[tool.poetry]
+[project]
+authors = [
+    { name = "Flavio Codeco Coelho", email = "fccoelho@gmail.com" },
+    { name = "Sandro Loch", email = "es.loch@gmail.com" },
+]
+license = { text = "AGPL-3.0" }
+requires-python = "<4,>=3.9"
+dependencies = [
+    "dbfread<3,>=2.0.7",
+    "tqdm<5,>=4.64.0",
+    "cffi<2,>=1.15.1",
+    "pyyaml>=6",
+    "setuptools>=75.6.0",
+]
 name = "pyreaddbc"
-version = "1.1.0"  # changed by semantic-release
+version = "1.2.0"
```

Comment:
please change this line back to `version = "1.1.0"  # changed by semantic-release`, it will be changed automatically when the release is triggered successfully